### ISSUE TITLE
Scroll threw player list and item lists

### ISSUE THREAD

**Issue by Ne-Row** [Oakraven79/pavlov_rcon_tcl#2]
I have a 1920x1080 monitor, so to see my whole list of players, its difficult to scroll through player list and item lists. Is it possible to make items list and player list scrollable so I can use my index's scroll, or mouse scroll to move through lists?

**Comment by Oakraven79** [comment]
Yep can do. Will need your help to test on the Index as i don't have one.

**Comment by Ne-Row** [comment]
Sure! Anytime!

**Comment by Oakraven79** [comment]
Release 0.69 ( https://github.com/Oakraven79/pavlov_rcon_tcl/releases/download/0.69/pavlov_rcon_tcl-0.69.zip ) addresses the player list. currently still working on the items list.
